spec/components/courses/international_students_component_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
 
 describe Courses::InternationalStudentsComponent do
-  context 'when the provider does not sponsor visa', type: :component do
-    it 'renders correct message' do
+  context 'when the course is fee-paying and the provider does not sponsor Student visas', type: :component do
```

Comment:
I don't think you need `type: :component` and the end of each context. I would put after the describe block at top of the file

Reply:
Thanks. That was already there. Doe it do anything?

Reply:
Ah yeah, it does. Moved to describe block in 0e13bfdb7b53c353a468362e5f77459f2dccbf99 - thanks.